src/components/30-organisms/footer/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.1.27
+
+- Fix: Now the height adapts automatically to the number of elements. #2142
```

Comment:
Your code below does not suppport this claim.

From what I can see, dynamic height adaptation only works for children defined under the specifics of the React rendering model.

This is far from general, and will not work for users that dynamically add or remove children using pure DOM API methods such as `appendChild`.

Now, this *may* be acceptable for the use case of the original issue that lead to this PR (is it?). If so, it should be properly reflected in a much more narrow claim that references React.

Reply:
Have changed the wording.